### PR TITLE
document reviewing the Ansible configuration

### PR DESCRIPTION
**Pull request by samccann**

### CODE REVIEW

File: downstream/assemblies/navigator/assembly-review-config-navigator.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+
+[id="assembly-review-config-navigator_{context}"]
+= Review your Ansible configuration with {Navigator}
+
+:context: review-config-navigator
+
+[role="_abstract"]
+
+As a content developer, you can review your Ansible configuration with {Navigator} and interactively delve into settings.
```
Comment (thaumos) [inline downstream/assemblies/navigator/assembly-review-config-navigator.adoc:12]: s/developer/creator

**Review by thaumos — APPROVED** [review]

File: downstream/modules/navigator/proc-review-config-tui.adoc [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+* A valid Ansible configuration file.
+
+.Procedure
+
+
+. Start {Navigator}
++
+```
+$ ansible-navigator
```
Comment (cidrblock) [inline downstream/modules/navigator/proc-review-config-tui.adoc:22]: or `ansible-navigator config` from the command line

File: downstream/modules/navigator/proc-review-config-tui.adoc [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+ 10│ANSIBLE_PIPELINING                   True default default                         False
+ 11│ANY_ERRORS_FATAL                     True default default                         False
+ 12│BECOME_ALLOW_SAME_USER               True default default                         False
+ 13│BECOME_PLUGIN_PATH                   True default default                         ['/home/runner/.ansible/plugins/become', '/usr/share/ansible/plugins/become']
+ 14│CACHE_PLUGIN                         False env    ANSIBLE_CACHE_PLUGIN            jsonfile
+ 15│CACHE_PLUGIN_CONNECTION              False env   ANSIBLE_CACHE_PLUGIN_CONNECTION /runner/artifacts/d2e5510d-b518-4625-8067-5a4da2a1d32c/fact_cac
+ 16│CACHE_PLUGIN_PREFIX                  True default default                         ansible_facts
+
+```
+
```
Comment (cidrblock) [inline downstream/modules/navigator/proc-review-config-tui.adoc:50]: maybe a note that some values reflect settings inside the execution environment necessary for the execution environment to function.  These will show as non-default, but may not be set in the user's ansible config file

File: downstream/modules/navigator/proc-review-config-tui.adoc [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+[id="proc-review-config-tui_{context}"]
+
+
+
+= Reviewing your Ansible configuration from the {Navigator} textual user interface
```
[the remark this answers is not in the document]
Comment (cidrblock) [reply downstream/modules/navigator/proc-review-config-tui.adoc:6]: if not, let me know and the repo can be updated to reflect these docs

**Review by cidrblock — CHANGES_REQUESTED** [review]

File: downstream/modules/navigator/proc-review-config-tui.adoc [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+ 11│ANY_ERRORS_FATAL                     True default default                         False
+ 12│BECOME_ALLOW_SAME_USER               True default default                         False
+ 13│BECOME_PLUGIN_PATH                   True default default                         ['/home/runner/.ansible/plugins/become', '/usr/share/ansible/plugins/become']
+ 14│CACHE_PLUGIN                         False env    ANSIBLE_CACHE_PLUGIN            jsonfile
+ 15│CACHE_PLUGIN_CONNECTION              False env   ANSIBLE_CACHE_PLUGIN_CONNECTION /runner/artifacts/d2e5510d-b518-4625-8067-5a4da2a1d32c/fact_cac
+ 16│CACHE_PLUGIN_PREFIX                  True default default                         ansible_facts
+
+```
+
+. Type the number corresponding to the setting you want to delve into, or type `:<number>` for numbers greater than 9.
```
Comment (cidrblock) [inline downstream/modules/navigator/proc-review-config-tui.adoc:51]: Do we have a navigation within Navigator doc where we can splain how to get around in it?

**Review by cidrblock — APPROVED** [review]

File: downstream/modules/navigator/proc-review-config-tui.adoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (samccann) [reply downstream/modules/navigator/proc-review-config-tui.adoc:6]: I switched to text-based user interface.

File: downstream/modules/navigator/proc-review-config-tui.adoc (hunk 5 above)
Comment (samccann) [reply downstream/modules/navigator/proc-review-config-tui.adoc:52]: not yet. I'll add it to the todo list.

**Pull request merged**